anvill/python/anvill/arch.py

```diff
@@ -48,7 +49,7 @@ def stack_offset(self) -> int:
         ...
 
     @abstractmethod
-    def register_family(self, reg_name) -> tuple:
+    def register_family(self, reg_name) -> Tuple[Tuple[str, ...]]:
```

Comment:
I think it's `Tuple[Tuple[str, int, int], ...]`?

Reply:
Maybe what you can do is assign something at the top level, e.g. `RegisterBounds = Tuple[Register, int, int]`, then you can do `-> Tuple[RegisterBounds, ...]`